code/plugins/authorizenet/authorizenet/tmpl/default.php

```diff
@@ -26,7 +26,6 @@ function myValidate(f)
 	{
```

Comment:
Why is this JS repeated in each layout ? It should be in a parent layout, and the sub-layouts should only have BS2/BS3 specific markup.